jobs/timeNearlyUpReminder.js

```diff
@@ -0,0 +1,53 @@
+var ServerEnv        = require('../models/serverEnvironment'),
+    notifier         = require('../services/notifier'),
+    jobConfig        = config.jobs.timeNearlyUpReminder,
+    timerId          = -1;
+
+var NEARLY_DONE_THRESHOLD = 10 * 60 * 1000; // 10 minutes
+
+module.exports = {
+
+    startJob: function() {
+        var that = this;
+
+        timerId = setInterval(function () {
+            console.log('Starting Reminder Job');
+
+            that.getNearlyDoneServers(function(servers) {
+                servers.forEach(function(server) {
+                    var usernameHandle = '@' + server.user.email.split('@',1)[0];
```

Comment:
you don't need to define var twice, put a comma between two vars coming after one another:
var a = 3,
      b = 4;